app/assets/stylesheets/scaffolds.scss

```diff
@@ -0,0 +1,65 @@
+body {
```

Comment:
Watch out!

These CSS definitions will affect the entire application
It's not a good practice to directly set properties to HTML elements. It's better to create classes  as much as possible